tests/integration/test_call_subqueries.py

```
@@ -0,0 +1,292 @@
+            """
+            CALL {
+                MATCH (p:Person) RETURN p.name AS person_name
+            }
+            CALL {
+                MATCH (c:Company) RETURN c.name AS company_name
+            }
+            RETURN person_name, company_name
+            """
+        )
+
+        # Cartesian product: 1 person × 1 company = 1 row
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)